### PR TITLE
ci(client): Revert try Jest's --onlyChanged for functional tests

### PR DESCRIPTION
**Pull request by danstarns**
Reverts: https://github.com/prisma/prisma/pull/15557

It is causing the client test suite to fail silently. 

Follow thread: https://prisma-company.slack.com/archives/C02FZENHD4N/p1664445150927359

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -76,7 +76,7 @@ jobs:
     steps:
       - uses: fkirc/skip-duplicate-actions@v5
         with:
-          cancel_others: 'true'
+          cancel_others: 'false'
```
Comment (aqrln) [inline .github/workflows/test.yml:79]: This comes from a different commit (https://github.com/prisma/prisma/commit/38318ed2544dc88d63f6d475354c24a2ef04caea). Is it intentional?

File: packages/client/helpers/functional-test/run-tests.ts [hunk 2]
```diff
@@ -34,12 +34,6 @@ const args = arg(
     '--no-mini-proxy-default-ca': Boolean,
     // Enable debug logs in the bundled Mini-Proxy server
     '--mini-proxy-debug': Boolean,
-    // Passes the same flag to Jest to only run tests related to changed files
```
Comment (aqrln) [inline packages/client/helpers/functional-test/run-tests.ts:37]: I think it's still nice to have support for these things locally

**Comment by Jolg42** [comment]
I made the one line change here instead https://github.com/prisma/prisma/pull/15609

File: packages/client/helpers/functional-test/run-tests.ts (hunk 2 above)
Comment (Jolg42) [reply packages/client/helpers/functional-test/run-tests.ts:37]: +1, I created a separate PR

**Pull request closed without merging**